webpack.cli.config.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'path';
+import { type Configuration } from 'webpack';
+import { rules } from './webpack.rules';
+
+const config: Configuration = {
+	target: 'node',
+	entry: './cli/index.ts',
+	output: {
+		filename: '[name].js',
+		path: path.join( __dirname, 'dist', 'cli' ),
+		libraryTarget: 'commonjs2',
+		chunkFormat: 'commonjs',
+	},
+	mode: process.env.NODE_ENV === 'production' ? 'production' : 'development',
+	devtool: 'source-map',
+	module: {
+		rules,
+	},
+	resolve: {
+		extensions: [ '.js', '.ts', '.json' ],
+		alias: {
+			cli: path.join( __dirname, 'cli' ),
+		},
+	},
+	optimization: {
+		minimize: false,
+		splitChunks: false,
+	},
+};
+
+export default config;
```

Comment:
Discussed previously but I'm not sure we need webpack for the CLI tool. running typescript directly should be enough at the moment.

Reply:
I discussed this in the PR description. Transpiling is enough to make it so node can run the code, but our build pipeline does not currently include a `node_modules` folder anywhere in the output, and we need that for the `require` algorithm to function normally. 

Bundling is an easy solution to this problem. The other option (that I see) is figuring out how to include a deduplicated/tree-shaken `node_modules` folder in the app output without increasing the installer size too much. 

I'm open to a different solution here down the line, but this seemed like a pragmatic approach for now.

Reply:
👍 You convinced me. Agreed it seems pragmatic, we can adapt later if needed.